app/javascript/components/server-components/Settings/AuthorizedApplicationsPage.tsx

```diff
@@ -43,7 +43,7 @@ const SCOPE_DESCRIPTIONS: Record<Scope, string> = {
   edit_products: "Create new products and edit your existing products.",
   ifttt: "See your sales data.",
   mark_sales_as_shipped: "Mark your sales as shipped.",
-  refund_sales: "Refund your sales.",
+  refund_sales: "Refund your sales and resend purchase receipts to customers.",
```

Comment:
I would update this to be edit_sales instead of refund_sales to match the enhanced functionality. It also means we can add even more stuff to it over time.